css/style.css

```diff
@@ -0,0 +1,2886 @@
+  }
+
+  .bottom__description {
+    width: 335px;
+    padding-left: 15px;
+  }
+
+  .bottom__stars {
+    margin-left: 15px;
+    padding-left: 0;
+    width: 350px;
+  }
+
+  .bottom__content {
+    width: 292px;
+    padding-left: 0;
+    margin-left: 0;
+  }
+    
+  .stars {
+    padding-right: 0;
+  }
+
+  .bottom__btn {
+    margin-left: 100px;
+  }
+
+}
+
+@media (max-width: 700px) {
+  .container {
+    max-width: 100%;
+  }
+
+  .top {
+    padding: 90px 0 30px;
+    padding-top: 26px;
+    background: url(../img/tesla-fon.png);
+    background-position: 0 160px;
+    background-repeat: no-repeat;
+    background-size: 700px 271px;
+  }
+
+  .top__wrapper-charset {
+    background: url(../img/svg/road-ipad.svg) no-repeat;
+    background-position: 370px 0;
+  }
+
+  .charset__speed {
+    width: 65.92px;
+    height: 50.16px;
+    padding-left: 0;
+    margin-left: 65px;
+  }
+
+  .middle__inner {
+    max-width: 700px;
+  }
+
+  .middle__fon {
+    max-width: 700px;
+  }
+
+  .middle__info-row {
+    max-width: 700px;
+  }
+
+  .middle__title {
+    margin-left: 300px;
+  }
+
+  .middle__row {
+    margin-right: 0;
+    margin-left: 40px;
+    padding-left: 0;
+    padding-right: 0;
+  }
+
+  .bottom {
+    background: url(../img/model-s-fon-ipad.png) no-repeat;
+    background-position: 55px 68px;
+  }
+
+  .bottom__inner {
+    background: url(../img/car-ipad.png) no-repeat;
+    background-size: 100%;
+    background-position: 0 110px;
+  }
+
+  .bottom__stars {
+    display: none;
+  }
+
+  .bottom__btn {
+    display: none;
+  }
+
+}
+
+@media (max-width: 650px) {
+  .container {
+    max-width: 100%;
+  }
+
+  .top {
+    padding: 90px 0 30px;
+    padding-top: 26px;
+    background: url(../img/tesla-fon.png);
+    background-position: 0 160px;
+    background-repeat: no-repeat;
+    background-size: 650px 271px;
+  }
+
+  .top__wrapper-charset {
+    background: url(../img/svg/road-ipad.svg) no-repeat;
+    background-position: 300px 0;
+  }
+
+  .top__wrapper-item {
+    margin-left: 0;
+  }
+
+  .charset {
+    margin-right: 0;
+  }
+
+  .charset__speed {
+    width: 65.92px;
+    height: 50.16px;
+    padding-left: 0;
+    margin-left: 50px;
+  }
+
+  .middle__inner {
+    max-width: 650px;
+  }
+
+  .middle__fon {
+    max-width: 650px;
+  }
+
+  .middle__info-row {
+    max-width: 650px;
+  }
+
+  .middle__title {
+    margin-left: 300px;
+  }
+
+  .middle__info-title {
+    font-size: 24px;
+  }
+
+  .middle__info-text {
+    font-size: 13px;
+    width: 280px;
+  }
+
+  .middle__row {
+    margin-left: 0;
+    padding-left: 0;
+    padding-right: 0;
+  }
+
+  .middle__btn {
+    margin-left: 90px;
+  }
+
+  .bottom {
+    background: url(../img/model-s-fon-ipad.png) no-repeat;
+    background-size: 550px 100px;
+    background-position: 38px 68px;
+  }
+
+  .bottom__inner {
+    background: url(../img/car-ipad.png) no-repeat;
+    background-size: 100%;
+    background-position: 0 130px;
+    }
```

Comment:
Codacy found an issue: [Expected indentation of 0 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/vovafinal/pullRequest?prid=5998857)